R/fiveyeargwsum-render.R

```diff
@@ -102,19 +102,6 @@ createfiveyeargwsumPlot <- function(reportObject){
 
   plot_object <- 
     XAxisLabels(plot_object, month_label, month_label_location, date_seq_yr + months(6))
-  
-  if(!isEmptyOrBlank(stat2TimeSeries)) {
```

Comment:
Is this stuff handled elsewhere now? Sorry, been a long time since I've looked at this code - just want to make sure I'm understanding what was removed here

Reply:
The plotting of the data is handled by calls below (now lines 106/111/116/121). These lines were setting up the sides and axes but they don't seem to be necessary at all - and were actually shifting the lims incorrectly to push the approval bars below the lower lims.  I noticed because there wasn't a similar line related to the primary time series/stat1TimeSeries, which was displaying approvals properly its own.. so removed these and then magically they all work again..

Reply:
Gotcha, okay makes sense. The only thing I can think of with these is that they might have been for implementing the 3rd y-axis stuff when the additional stat derived series are on other parameters? But as long as that still works I think this looks good :+1:

Reply:
yeah, i think it may have been necessary at some point, maybe prior to a gsplot change? but now it isn't. but here's a screenshot of this code with 4 diff params

![image](https://user-images.githubusercontent.com/6920344/46752458-df5c8f00-cc82-11e8-910b-ad8b169f7c45.png)

Reply:
Awesome! 🎨